examples/discovery-search-app/cypress/fixtures/basic/noResults.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
So, organize the fixtures based on the path or endpoint that each request is hitting? I could get behind that. It might make it easier to tell what fixtures can be reused where. We will have a lot of fixtures in the `/query` directory, though.